README.md

```diff
@@ -85,22 +88,22 @@ Uptime Kuma is now running on http://localhost:3001
 More useful PM2 Commands
 
 ```bash
-# If you want to see the current console output
+# If you want to see the current console output run:
 pm2 monit
 
-# If you want to add it to startup
+# If you want to start uptime kuma when PM2 is started run:
 pm2 save && pm2 startup
 ```
 
 ### Advanced Installation
 
-If you need more options or need to browse via a reverse proxy, please read:
+If you need more installation options or need to use Uptime Kuma over a reverse proxy, please read the <a href="https://github.com/louislam/uptime-kuma/wiki/%F0%9F%94%A7-How-to-Install">installation guide.</a>
 
 https://github.com/louislam/uptime-kuma/wiki/%F0%9F%94%A7-How-to-Install
```

Comment:
You did not remove this link, even though you removed the `:` in the last line

Reply:
Yes i did not.